client/__test__/actions/readbyAction.test.jsx

```diff
@@ -0,0 +1,8 @@
+import expect from 'expect';
+import updateReadBy from '../../actions/readbyAction';
+
+describe('Message Readby Action', () => {
+  it('should return a function', () => {
```

Comment:
I like the correlation of your test descriptive messages but can you be more specific to say what the function that is returning actually does, for example, _Message Ready Action should dispatch Message Ready action type with payload_